<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GomokuWindow</class>
 <widget class="QMainWindow" name="GomokuWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1081</width>
    <height>850</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gomoku</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="0">
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="menu">
       <layout class="QGridLayout" name="gridLayout_3">
        <property name="sizeConstraint">
         <enum>QLayout::SetNoConstraint</enum>
        </property>
        <item row="1" column="0" colspan="2">
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLabel" name="menuTitle">
            <property name="font">
             <font>
              <family>Bandal</family>
              <pointsize>44</pointsize>
              <weight>50</weight>
              <bold>false</bold>
              <underline>false</underline>
             </font>
            </property>
            <property name="text">
             <string>Gomoku</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignHCenter|Qt::AlignTop</set>
            </property>
            <property name="margin">
             <number>25</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="4" column="0" rowspan="3" colspan="2">
         <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,5,1">
          <property name="sizeConstraint">
           <enum>QLayout::SetNoConstraint</enum>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <layout class="QGridLayout" name="gridLayout_2">
            <property name="sizeConstraint">
             <enum>QLayout::SetNoConstraint</enum>
            </property>
            <item row="2" column="0">
             <widget class="QPushButton" name="exitButton">
              <property name="text">
               <string>Exit</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QPushButton" name="IAButton">
              <property name="text">
               <string>Player vs Dolores</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QPushButton" name="pvpButton">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>0</height>
               </size>
              </property>
              <property name="text">
               <string>Player vs Player</string>
              </property>
              <property name="iconSize">
               <size>
                <width>16</width>
                <height>16</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="gameRoom">
       <layout class="QGridLayout" name="_2">
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>36</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <property name="spacing">
         <number>0</number>
        </property>
        <item row="1" column="3">
         <layout class="QGridLayout" name="gridLayout_4" rowstretch="0" columnstretch="2,10" rowminimumheight="0" columnminimumwidth="0,0">
          <property name="sizeConstraint">
           <enum>QLayout::SetMinAndMaxSize</enum>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <property name="spacing">
           <number>0</number>
          </property>
          <item row="0" column="0">
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <property name="topMargin">
             <number>107</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>120</number>
            </property>
            <item>
             <widget class="QLabel" name="playerTurn">
              <property name="font">
               <font>
                <family>Bandal</family>
                <pointsize>20</pointsize>
                <weight>50</weight>
                <bold>false</bold>
                <underline>false</underline>
                <strikeout>false</strikeout>
                <kerning>false</kerning>
               </font>
              </property>
              <property name="text">
               <string>White Turn</string>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
               <widget class="QLabel" name="label">
                <property name="text">
                 <string>Double Trois</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="doubleTrois">
                <property name="text">
                 <string/>
                </property>
                <property name="checked">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <item>
               <widget class="QLabel" name="label_2">
                <property name="text">
                 <string>Cinq Cassable</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="cinqCassble">
                <property name="text">
                 <string/>
                </property>
                <property name="checked">
                 <bool>true</bool>
                </property>
                <property name="tristate">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QLabel" name="time">
              <property name="text">
               <string>Time : 0</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="turn">
              <property name="text">
               <string>Turn : 0</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="whiteScore">
              <property name="text">
               <string>White : </string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="blackScore">
              <property name="text">
               <string>Black : </string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="0" column="1">
           <widget class="QFrame" name="sfmlFrame">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout"/>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1081</width>
     <height>19</height>
    </rect>
   </property>
   <property name="nativeMenuBar">
    <bool>true</bool>
   </property>
   <widget class="QMenu" name="menuGame">
    <property name="title">
     <string>Game</string>
    </property>
    <addaction name="actionReplay"/>
    <addaction name="actionBack_to_the_menu"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuGame"/>
  </widget>
  <action name="actionReplay">
   <property name="text">
    <string>Replay</string>
   </property>
  </action>
  <action name="actionBack_to_the_menu">
   <property name="text">
    <string>Back to the menu</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
